packages/menu/src/test/mwc-menu.test.ts

```diff
@@ -0,0 +1,307 @@
+/**
+ * @license
+ * Copyright 2020 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import '@material/mwc-list/mwc-list-item';
+
+import {List} from '@material/mwc-list';
+import {ListItem} from '@material/mwc-list/mwc-list-item';
+import {Menu} from '@material/mwc-menu';
+import {MenuSurface} from '@material/mwc-menu/mwc-menu-surface';
+
+import {Fake, rafPromise} from '../../../../test/src/util/helpers';
+
+suite('mwc-menu', () => {
```

Comment:
Amazing work! Saved so much time here! Missing cases are:

* clicking on a list item
* multi-select menu with grouped list items [see grouped menu demo](https://mwc-demos.glitch.me/demos/menu.html) (and subsequently clicking and checking selected)

Feel free to file a bug if you don't want to implement